README.md

```diff
@@ -50,18 +50,30 @@ port=8888                                         # Replace this by the port on
 
 ## Requirements
 
-Operating System: Currently only Linux and Windows are supported.
+ - Operating System: Currently only Linux and Windows are supported.
+ - Python extension (ms-python.python) installed
 
 ## Extension Settings
 
 You can find the settings of the extension, in the Settings panel, under `Extensions > Inmanta`.
 
 ![Settings screenshot](images/screenshot-settings.png)
 
-The extension currently has three settings:
- - inmanta.compilerVenv: Absolute path to the virtual environment the compiler should use.
+The extension currently has two settings:
+ - inmanta.compilerVenv: Absolute path to the virtual environment the compiler and language server should use.
  - inmanta.ls.enabled: Whether or not activate the Inmanta Language server (you probably want to).
- - inmanta.pythonPath: Python path the extension should use to run the language server.
+
+### Python environments
+The plugin uses the Python extension to change environments. This can be done by clicking on the active environment in the bottombar
+and then selecting the environment to activate.
+
+<img src="images/python-extension.png" width="40%">
```

Comment:
@arnaudsjs You are talking about the other image right?

Reply:
Yes. Indeed.

Reply:
I changed the image a bit. Do you find it less misleading?

Reply:
The fact that the two boxes at the bottom are stacked on top of each other, is the element I find most confusing. If I understand everything correctly, they are mutually exclusive. So I would expect something like this:

```
                         ----------------
                         | Extension |
                         ----------------
         (inmanta-core>=6)  /           \  (inmanta-core<6)
                           /             \
               -----------------       -----------------------
               | Active env |       | Compiler Venv |
               -----------------       -----------------------
```

Do you agree? (It could be that I am missing something)

Also: The image mentions modules V2 for the case where `inmanta-core<6`, but `inmanta-core<6` doesn't support modules V2.

Reply:
> Also: The image mentions modules V2 for the case where inmanta-core<6, but inmanta-core<6 doesn't support modules V2.

Should it be V1 modules? 
(I took the image from the presentation sander did last Friday :p )

Reply:
I see. I considered the installation of the language server a part of the top box. But it's probably better like it is now. I think two question remain here:

* Why is `compilerVenv option` mentioned on the arrow between the first and the second box from the top? As far as I can see, that option is only relevant when a `CompilerVenv` is used (bottom box).
* Regarding your question about V2 module: Currently we support V1 modules on `inmanta-core<6` and we support V1 and V2 modules on `inmanta-core>=6`, but only V2 modules are installed as a Python package in a venv. V1 modules are installed by performing a Git checkout in a certain directory. The Python dependencies of a V1 or a V2 module are installed a the venv. I think that's what `module dependencies` means on the diagram. Conclusion: I think we should remove `V2 modules` from the bottom box.

Reply:
- The compilerVenv option is mentioned on the arrow  between the first and the second box because the extension will always send it to the LS(second box). It is the LS that will decide if he will use the option or not. Maybe it is misleading as the user might not really need to know this? 
 - i'll make the change on the image once we agree about everything :p

Reply:
Regarding bullet 1: I agree that this is an implementation detail that is irrelevant for end-users.

Reply:
I changed the image :)

Reply:
Looks good!